client/packages/common/src/ui/layout/tables/material-react-table/useTableFiltering.ts

```diff
@@ -108,28 +93,30 @@ export const useManualTableFilters = <T extends MRT_RowData>(
   };
 
   return {
-    filterState,
-    filterHandlers: { onColumnFiltersChange: handleFilterChange },
+    columnFilters: filterState,
+    onColumnFiltersChange: handleFilterChange,
```

Comment:
this hook is a little simpler now, that we always sort the filter state in the URL - the `manualFiltering` only controls whether table will use its internal filter methods, or expect filtered data to be passed in